track.go

```diff
@@ -196,7 +196,9 @@ func (track *baseTrack) bind(ctx webrtc.TrackLocalContext, specializedTrack Trac
 		var doneCh chan<- struct{}
 		writer := ctx.WriteStream()
 		defer func() {
-			close(stopRead)
+			if stopRead != nil {
```

Comment:
I'm not seeing where the channel is ever not initialized or explicitly set to nil. Can you point me at that or provide a reproduction of the panic you mention?